components/wrist.py

```diff
@@ -117,6 +122,10 @@ def inclination(self) -> float:
     def inclination_deg(self) -> float:
         return math.degrees(self.inclination())
 
+    @feedback
+    def test_inclination(self) -> float:
+        return math.degrees(self.inclination() - self.COM_DIFFERENCE)
+
```

Comment:
This is poorly named for what it does. Call this `link_inclination_deg`. the `test_` prefix doesn't provide any value and following our current naming convention I expect this to return radians.